src/libraries/Microsoft.PowerFx.Core/Public/Values/UntypedObjectValue.cs

```diff
@@ -43,7 +43,13 @@ public interface IUntypedObject
         // It is validated to be in a culture invariant standard format number, possibly with dot decimal separator and "E" exponent.
         // This method need not be implemetned if ExteranlType.UntypedNumber is not used.
         // GetDouble and GetDecimal can be called on an ExternalType.UntypedNumber.
-        string GetUntypedNumber();
+        string GetUntypedNumber();

```

Comment:
>GetUntypedNumber [](http://example.com/codeflow?start=15&length=16)

I might name it GetRawValue() to avoid any confusion with *Untyped*Object

Reply:
let's do it in a separate cleanup pr